### DISCUSSION TITLE
Having a problem when upoading to google drive

### DISCUSSION THREAD

**Discussion by Geek-DB** [anasty17/mirror-leech-telegram-bot#1272] [Q&A]
Hi i'm having this error when the bot try to upload to google drive
EOF occurred in violation of protocol (_ssl.c:2396)
Plz help if possible

**Comment by anasty17** [comment]
Google it. Its not code issue. Maybe ur ip is blocked by some trackers or peers.

**Comment by ghost** [comment]
> Hi i'm having this error when the bot try to upload to google drive EOF occurred in violation of protocol (_ssl.c:2396) Plz help if possible

Got same problem, but not code issue
